src/main/java/bio/terra/app/controller/SearchApiController.java

```diff
@@ -96,6 +98,20 @@ private AuthenticatedUserRequest getAuthenticatedInfo() {
     }
   }
 
+  @Override
+  public ResponseEntity<SearchMetadataModel> upsertSearchMetadata(
+      @PathVariable("id") UUID id,
+      @Valid @RequestBody SearchMetadataRequest searchMetadataRequest) {
+    try {
+      SearchMetadataModel searchMetadataModel = new SearchMetadataModel();
+      searchMetadataModel.setMetadataSummary(
+          "Upserted search metadata for snapshot " + id.toString());
```

Comment:
`toString()` isn't needed in a string concat expression
```suggestion
          "Upserted search metadata for snapshot " + id);
```